src/main/java/co/org/osso/api/DefaultLogger.java

```diff
@@ -0,0 +1,29 @@
+package co.org.osso.api;
+
+import org.springframework.stereotype.Component;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.sql.Timestamp;
+
+@Component
+public class DefaultLogger implements Logger {
+
+    @Override
+    public void log(String message) {
+        Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+        InetAddress ip = null;
+        long threadId = Thread.currentThread().getId();
+
+        try {
+            ip = InetAddress.getLocalHost();
```

Comment:
How about we cache this value instead? Say, check if we can get it in the constructor. If we can, great! if not just don't print it instead of printing a stacktrace for each log.